Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/transform/audit/AdminDistTransforms.java

```diff
@@ -178,7 +179,11 @@ protected String getHomeCommunityFromMapping() {
     }
 
     private String getRelatesTo(AssertionType assertion) {
-        return NullChecker.isNotNullish(assertion.getRelatesToList()) ? assertion.getRelatesToList().get(0) : null;
+        return NullChecker.isNotNullish(assertion.getRelatesToList()) ? getRelatesToValue(assertion.getRelatesToList())
+            : null;
     }
 
+    private String getRelatesToValue(List<String> relatesTo) {
+        return NullChecker.isNotNullish(relatesTo.get(0)) ? relatesTo.get(0) : null;
```

Comment:
Just for  clarification. 

For RelatesTo if we pass blank RelatesTo Id from Soap Requests the Nwhin Outbound message does not have RealtesTo in the Header. That's the reason the audit logging is returning a Null value.

Can you confirm this?